### PR TITLE
Migrate Automation docs to .mdx

### PR DESCRIPTION
**Pull request by thedriftofwords**

### CODE REVIEW

**Comment by thedriftofwords** [comment]
Note: Submit PR #1201 first, then rebase.

File: src/pages/getting-started/intermediates-tutorial.mdx [hunk 1]
```diff
@@ -333,8 +337,10 @@ contract VRFD20 is VRFConsumerBaseV2 {
 ```
 
 <div class="remix-callout">
-  <a href="https://remix.ethereum.org/#url=https://docs.chain.link/samples/VRF/VRFD20.sol" target="_blank" >Open in Remix</a>
-  <a href="/getting-started/conceptual-overview/#what-is-remix" >What is Remix?</a>
+  <a href="https://remix.ethereum.org/#url=https://docs.chain.link/samples/VRF/VRFD20.sol" target="_blank">
+    Open in Remix
+  </a>
+  <a href="/getting-started/conceptual-overview/#what-is-remix">What is Remix?</a>
```
Comment (dwightjl) [inline src/pages/getting-started/intermediates-tutorial.mdx:343]: This Remix button has CSS issues. This one might be fixed by putting the previous code sample into the code sample component, which has a Remix button by default.

**Review by dwightjl — CHANGES_REQUESTED** [review]

**Comment by dwightjl** [comment]
Looks like this PR includes duplicate changes with https://github.com/smartcontractkit/documentation/pull/1201/files

Should we close that other PR and just work off of this one?

**Comment by thedriftofwords** [comment]
Note: Originally I planned to submit PR #1201 with the getting-started changes first, then rebase this one and submit. Now I've rebased this one to remove the getting-started commit so that I can submit it right away, while I try to figure out the issues you pointed out affecting the getting-started pages.

**Review by dwightjl — APPROVED** [review]

**Pull request merged**